### ISSUE TITLE
Massively simplify and clean up UI

### ISSUE THREAD

**Issue by jni** [microscopium/microscopium-ui#36]
I think there's a clear winner between the different panels — the neighbour nebula. I think, in fact, that our entire UI could be folded into it, as follows:

- [ ] the "Cluster cosmologist" can be a toggle on colouring of the dots in the very same plot, and a slider that is hidden most of the time (or at the very bottom).
- [x] the "Feature explorer" panels can be massively reduced and placed under the PCA plot
- [ ] the "Screen info" panel can just be a tooltip or a drawer, represented by an (?) or somesuch icon

Once this is done, the splash page can be either a screen selection splash or (if the user only has access to a single screen) the nebula UI directly!

This would really simplify and clean up the experience for new users...

**Comment by jni** [comment]
There's a 'first phase' of this that can be achieved very quickly, in addition to point 3 above:

- [ ] remove the title tabs on the top, just have a main tab, "microscopium", and relegate everything else to a drop-down.
- [ ] Launch into a screen selection splash screen, rather than a blank screen; if there's only one screen in the DB go straight to the main window.
- [ ] Go straight into the main window after selecting a screen

**Comment by starcalibre** [comment]
#45 implements this! Good to close this one?

**Comment by jni** [comment]
Indeed!
